content/razor-views/reading/view-form-walkthrough/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Walkthrough: Begin CodingEvents"
+date: 2023-02-09T12:48:24-06:00
+draft: false
+weight: 11
+originalAuthor: Courtney Frey # to be set by page creator
+originalAuthorGitHub: speudusa # to be set by page creator
+reviewer: Kimberly Horan # to be set by the page reviewer
+reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+Welcome to the Walkthrough for creating and using templates in your `CodingEvents` project.
+
+
+The video below is a walkthrough of the steps covered in the reading.  Time stamps have been added to highlight the topics covered so far and their locations in the reading.
+
+The following topics are covered in the walkthrough:
+- Creating an MVC project in Visual Studio 
+- Creating an `EventsController` that contains `Index`, `Add`, and `NewEvent` action methods
+- Creating an `Events` View directory that has `Index` and `Add` views.
+- Using `Redirect` to return a user to a view that contains an updated list of events.
+- Using [anchor tags](https://learn.microsoft.com/en-us/aspnet/core/mvc/views/tag-helpers/built-in/anchor-tag-helper?view=aspnetcore-6.0)
+
+## Video Walkthrough
+{{< youtube B_Uw_BpATjk >}}
```

Comment:
Update video link and add chapters to video

Reply:
check check.